### ISSUE TITLE
new relic widget_billboard (newrelic_one_dashboard) tresholds value 0 not set

### ISSUE THREAD

**Issue by arnehaese** [newrelic/terraform-provider-newrelic#1323]
Hi there,

Thank you for opening an issue. In order to better assist you with your issue, we kindly ask to follow the template format and instructions. Please note that we try to keep the Terraform issue tracker reserved for **bug reports** and **feature requests** only. General usage questions submitted as issues will be closed and redirected to New Relic's Explorers Hub https://discuss.newrelic.com/c/build-on-new-relic/developer-toolkit.

## Please include the following with your bug report

> :warning: **Important:** Failure to include the following, such as omitting the Terraform configuration in question, may delay resolving the issue.

- [ v2.22.1] Your New Relic `provider` [configuration](#terraform-configuration-files) (sensitive details redacted)
- [x] A list of [affected resources](#affected-resources) and/or data sources
- [ ] The [configuration](#terraform-configuration-files) of the resources and/or data sources related to the bug report (i.e. from the list mentioned above)
- [ ] Description of the [current behavior](#actual-behavior) (the bug)
- [ ] Description of the [expected behavior](#expected-behavior)
- [ ] Any related [log output](#debug-output)

### Terraform Version
Run `terraform -v` to show the version. If you are not running the latest version of Terraform, please upgrade because your issue may have already been fixed.
Terraform v0.15.1
newrelic/newrelic v2.22.1

### Affected Resource(s)
Please list the resources as a list, for example:
- resource "newrelic_one_dashboard" 

### Terraform Configuration
> Please include your `provider` configuration (sensitive details redacted) as well as the configuration of the resources and/or data sources related to the bug report.
```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.
widget_billboard {
      title = "Jenkins failed jobs"
      row = 7
      column = 11
      width         = 2
      height        = 3
      warning       = 0
      critical      = 1
      
      nrql_query {
        query       = "SELECT uniqueCount(jobName) FROM AppBuildEvent WHERE buildResult = 'FAILURE' SINCE 24 hours ago EXTRAPOLATE"
      }
    }

```

### Actual Behavior
What actually happened?
When the dashboard is created, the treshold for warning is not enabled (empty) for the widget - if I manually set it to 0 in the edit gui of new relic it is saved as a 0 hence reporting my value of on jenkins job failing as warning in yellow as expected - the treshold for critical is set to 1 correctly.

### Expected Behavior
What should have happened?
I expect the 0 value to be set for the warning treshold

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform apply`

### Debug Output
Please provider a link to a GitHub Gist containing the complete debug output: https://www.terraform.io/docs/internals/debugging.html. Please do NOT paste the debug output in the issue; just paste a link to the Gist.

### Panic Output
If Terraform produced a panic, please provide a link to a GitHub Gist containing the output of the `crash.log`.

### Important Factoids
Are there anything atypical about your accounts that we should know? For example: Running in EC2 Classic? Custom version of OpenStack? Tight ACLs?

### References
Are there any other GitHub issues (open or closed) or Pull Requests that should be linked here? For example:
- GH-1234

**Comment by kidk** [comment]
Thanks for reporting this @arnehaese 

We'll work on getting this resolved.

**Comment by jthurman42** [comment]
This is most certainly an "unset vs no value" issue, probably [here](https://github.com/newrelic/newrelic-client-go/blob/main/pkg/dashboards/types.go#L91) in `newrelic-client-go`.  Would need to update `Value` to be a `*float64` to determine set vs unset, or remove `omitempty` from the json tag.

**Comment by mbazhlekova** [comment]
Hey @arnehaese 👋 we released a fix to this bug with version 2.24.1
